packages/astrocite-bibtex/src/parser.ts

```diff
@@ -123,14 +125,26 @@ const parseEntry = (node: Entry, macros: Map<string, string>): Data => {
                         .map(parseName),
                 };
                 break;
-            default:
+            }
+            case 'number': {
+                // map "number" to "issue" unless there's an "issue" field
+                const field = keys.includes('issue') ? 'number' : 'issue';
+                entry = {
+                    ...entry,
+                    [field]: parseValue(property.value, macros),
+                };
+                break;
+            }
+            default: {
                 const field = FIELD_MAP.get(property.key);
                 if (field) {
                     entry = {
                         ...entry,
                         [field]: parseValue(property.value, macros),
                     };
                 }
+                break;
```

Comment:
`break` isn't necessary in the default case.

```suggestion
```

Reply:
Just gonna commit this suggestion now without your approval since it's so trivial and because i'd like to get this pushed out now so I don't forget. Hope that's alright. @hubgit